flow/connectors/bigquery/qvalue_convert.go

```diff
@@ -0,0 +1,74 @@
+package connbigquery
+
+import (
+	"fmt"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/PeerDB-io/peer-flow/model/qvalue"
+)
+
+func qValueKindToBigQueryType(colType string) bigquery.FieldType {
+	switch qvalue.QValueKind(colType) {
+	// boolean
+	case qvalue.QValueKindBoolean:
+		return bigquery.BooleanFieldType
+	// integer types
+	case qvalue.QValueKindInt16, qvalue.QValueKindInt32, qvalue.QValueKindInt64:
+		return bigquery.IntegerFieldType
+	// decimal types
+	case qvalue.QValueKindFloat32, qvalue.QValueKindFloat64:
+		return bigquery.FloatFieldType
+	case qvalue.QValueKindNumeric:
+		return bigquery.NumericFieldType
+	// string related
+	case qvalue.QValueKindString:
+		return bigquery.StringFieldType
+	// json also is stored as string for now
+	case qvalue.QValueKindJSON:
+		return bigquery.StringFieldType
+	// time related
+	case qvalue.QValueKindTimestamp, qvalue.QValueKindTimestampTZ:
+		return bigquery.TimestampFieldType
+	case qvalue.QValueKindDate:
+		return bigquery.DateFieldType
+	case qvalue.QValueKindTime, qvalue.QValueKindTimeTZ:
+		return bigquery.TimeFieldType
+	// TODO: handle INTERVAL types again
```

Comment:
link the issue in all the places in code as well, something like
`TODO (<issue_url>): description`

Reply:
we don't support `DATE` as well right?